compiler/main/DynFlags.hs

```diff
@@ -4053,6 +4054,7 @@ wWarningFlagsDeps = [
   flagSpec "star-is-type"                Opt_WarnStarIsType,
   flagSpec "missing-space-after-bang"    Opt_WarnSpaceAfterBang,
   flagSpec "controller-can"              Opt_WarnControllerCan,
+  flagSpec "unsupported-daml-names"      Opt_WarnUnsupportedDamlNames,
```

Comment:
nitpick, I think this should be `unsupported-daml-field-names` so intent is a bit clearer

Reply:
Ah right, my intent was that the `daml` part was single it down to the specific daml fields, but yeah this is more clear